### PR TITLE
fix(property-filter): use a subscription to update the page title

### PR DESCRIPTION
**Pull request by mariusandra**
## Problem

Do not ask me how, but this fixes the following.

## Changes

With this fix:

On master:

## How did you test this code?

Eyes closed.

### CODE REVIEW

File: frontend/src/layout/navigation/Breadcrumbs/breadcrumbsLogic.tsx [hunk 1]
```diff
@@ -146,5 +146,25 @@ export const breadcrumbsLogic = kea<breadcrumbsLogicType>({
...
+                        .reverse(),
+                    'PostHog',
+                ]
+            },
+        ],
+    }),
+
+    subscriptions: ({}: breadcrumbsLogicType) => ({
+        titleBreadcrumbs: (titleBreadcrumbs: Breadcrumb[]) => {
+            if (typeof document !== 'undefined') {
```
Comment (EDsCODE) [inline frontend/src/layout/navigation/Breadcrumbs/breadcrumbsLogic.tsx:165]: Maybe this solved it? Otherwise it seems like the logic would be the same

**Review by EDsCODE — APPROVED** [review]
I'm also not 100% certain why moving the logic here solves it but it fixes the bug and keeps more logic in kea so I'm ok with this

**Review by Twixes — CHANGES_REQUESTED** [review]

**Comment by mariusandra** [comment]
@Twixes Your hunch might be right, though it's still weird how this happens. It's possible adding another selector and using `[stringTitle]` for `useEffect` might have solved things. However I was too committed, and refactored this further... and added a test.

Unfortunately the `sceneBreadcrumbs` selector is recalculated way too often. Somebody also [requested a fix](https://github.com/keajs/kea/issues/143) for this. For us, for now, just having `documentTitle` a string seems to work.

**Review by Twixes — APPROVED** [review]
Looks proper 👌

**Pull request merged**